### DISCUSSION TITLE
Windows explorer integration hotkey is now "M"?

### DISCUSSION THREAD

**Discussion by jez9999** [WinMerge/winmerge#1891] [General]
In an older version of Winmerge, my Windows explorer integration popup menu hotkey for Winmerge was "W".  It now seems to have changed to "m" which is really annoying as it clashes with "Rename", which I use the hotkey for all the time.  Could Winmerge add an option to choose what hotkey is used for Winmerge, or at least choose between "W" and "M"?

**Comment by sdottaka** [comment]
W conflicts with "w" in "Ne&w", so I changed it to M ("&Move to OneDrive" or "Rena&me") where the conflict occurred by default.

As a workaround, uncheck the Add to context menu in the Options window as shown below, and the WinMerge menu will no longer appear.
You can also rename files and folders with the F2 key.

![image](https://github.com/WinMerge/winmerge/assets/98126/714cd884-811c-4a30-a4fe-6f6364a2397f)
